docs/integrations/azure/discovery.md

```diff
@@ -6,25 +6,78 @@ sidebar_label: Discovery
 description: Automatically discovering catalog entities from repositories in an Azure DevOps organization
 ---
 
-The Azure DevOps integration has a special discovery processor for discovering
-catalog entities within an Azure DevOps. The processor will crawl the Azure
+The Azure DevOps integration has a special entity provider for discovering
+catalog entities within an Azure DevOps. The provider will crawl your Azure
 DevOps organization and register entities matching the configured path. This can
 be useful as an alternative to static locations or manually adding things to the
 catalog.
 
 ## Installation
 
-You will have to add the processors in the catalog initialization code of your
-backend. They are not installed by default, therefore you have to add a
-dependency to `@backstage/plugin-catalog-backend-module-azure` to your backend
-package.
+At your configuration, you add one or more provider configs:
+
+```yaml
+# app-config.yaml
+catalog:
+  providers:
+    azureDevOps:
+      yourProviderId: # identifies your dataset / provider independent of config changes
+        organization: myorg
+        project: myproject
+        repository: service-* # this will match all repos starting with service-*
+        path: /catalog-info.yaml
+      anotherProviderId: # another identifier
+        organization: myorg
+        project: myproject
+        repository: '*' # this will match all repos starting with service-*
+        path: /src/*/catalog-info.yaml # this will search for files deep inside the /src folder
+      yetAotherProviderId: # guess, what? Another one :)
+        host: selfhostedazure.yourcompany.com
+        organization: myorg
+        project: myproject
+```
+
+The parameters available are:
+
+- `host:` Leave empty for Cloud hosted, otherwise set to your self-hosted instance host.
+- `organization:` Your organization slug. Required.
```

Comment:
For on-premise users there is no Organization it's a Collection, would be good to mention that: "Your Organization (Collection for on-premise users) slug. Required."